### DISCUSSION TITLE
Add docker and docker-compose file

### DISCUSSION THREAD

**Discussion by dassudip2001** [avelino/awesome-go#5038] [Q&A] [answered]
Can I add local devlopment using docker and docker-compose  or it is already present

**Comment by mounor** [comment]


**Reply by KingXnazim881** [reply]
job_spec [&]            history [-c] [-d off>
 (( expression ))        if COMMANDS; then CO>
 . filename [arguments>  jobs [-lnprs] [jobsp>
 :                       kill [-s sigspec | ->
 [ arg... ]              let arg [arg ...]
 [[ expression ]]        local [option] name[>
 alias [-p] [name[=val>  logout [n]
 bg [job_spec ...]       mapfile [-d delim] [>
 bind [-lpsvPSVX] [-m >  popd [-n] [+N | -N]
 break [n]               printf [-v var] form>
 builtin [shell-builti>  pushd [-n] [+N | -N >
 caller [expr]           pwd [-LP]
 case WORD in [PATTERN>  read [-ers] [-a arra>
 cd [-L|[-P [-e]] [-@]>  readarray [-d delim]>
 command [-pVv] comman>  readonly [-aAf] [nam>
 compgen [-abcdefgjksu>  return [n]
 complete [-abcdefgjks>  select NAME [in WORD>
 compopt [-o|+o option>  set [-abefhkmnptuvxB>
 continue [n]            shift [n]
 coproc [NAME] command>  shopt [-pqsu] [-o] [>
 declare [-aAfFgiIlnrt>  source filename [arg>
 dirs [-clpv] [+N] [-N>  suspend [-f]
 disown [-h] [-ar] [jo>  test [expr]
 echo [-neE] [arg ...>   time [-p] pipeline
 enable [-a] [-dnps] [>  times
 eval [arg ...]          trap [-lp] [[arg] si>
 exec [-cl] [-a name] >  true
 exit [n]                type [-afptP] name [>
 export [-fn] [name[=v>  typeset [-aAfFgiIlnr>
 false                   ulimit [-SHabcdefikl>
 fc [-e ename] [-lnr] >  umask [-p] [-S] [mod>
 fg [job_spec]           unalias [-a] name [n>
 for NAME [in WORDS ..>  unset [-f] [-v] [-n]>
 for (( exp1; exp2; ex>  until COMMANDS; do C>
 function name { COMMA>  variables - Names an>
 getopts optstring nam>  wait [-fn] [-p var] >
 hash [-lr] [-p pathna>  while COMMANDS; do C>
 help [-dms] [pattern >  { COMMANDS ; }
~ $

**Comment by avelino** [comment, marked as answer]
@dassudip2001 what would be the point of having dockerfile (and compose) in the project?

the Go codes run in isolation, see the workflows here
https://github.com/avelino/awesome-go/tree/main/.github/workflows
